api/base/serializers.py

```diff
@@ -1,16 +1,33 @@
 import re
 import collections
 
-from rest_framework.reverse import reverse
 from rest_framework.fields import SkipField
+from rest_framework.exceptions import NotFound
+from rest_framework.reverse import reverse
 from rest_framework import serializers as ser
 
+from framework.auth.core import Auth
+from framework.guid.model import Guid
 from website import settings
+from website.project.model import Node, Comment
 from website.util.sanitize import strip_html
 from website.util import waterbutler_api_url_for
```

Comment:
According to Steve, importing the module is generally preferred but it is a more flexible guideline.